docs/index.md

```diff
@@ -19,18 +19,31 @@ GitHub integration is made through the GitHub app interface and the checks API (
 which allows results to be presented directly as inline annotations instead of
 a pass/fail status report.
 
-Precaution currently supports analysis of python files via Bandit and go files via Gosec. New languages may be added in future.
+Precaution currently supports analysis of:
+* Go files via [Gosec](https://github.com/securego/gosec)
+* JavaScript and TypeScript via [TSLint](https://github.com/palantir/tslint) and [tslint-config-security](https://github.com/webschik/tslint-config-security)
+* Python files via [Bandit](https://github.com/PyCQA/bandit)
+
+New languages may be added in future, please file an [issue](https://github.com/vmware/precaution/issues) for your language/linter of choice.
 
 * Documentation: [vmware/precaution/docs](https://vmware.github.io/precaution/)
 * Source: [vmware/precaution](https://github.com/vmware/precaution)
-* Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues)
+<!-- * Bugs: [vmware/precaution/issues](https://github.com/vmware/precaution/issues) -->
```

Comment:
A general remark; don't use code comments to remove lines of code, just delete them. The revision control history (git) is able to show us what removed and, when paired with a decent commit message, why.